_posts/2023-02-16-secure-cloud-foundation.md

```diff
@@ -0,0 +1,275 @@
+
+<img src="{{ '/img/2023-02-16-secure-cloud-foundation/janitor.png' | prepend: site.baseurl }}" alt="Gandalf, You shall not pass" class="image fit" style="margin:0px auto; max-width:100%; height:60%;">
+
+Cloud Custodian uses a policy-driven approach to scan cloud resources. Policies are written using a simple and flexible policy language that allows organizations to define and manage policies.
+
+The features of Cloud Custodian for a secure cloud environment:
+
+### Automated Remediation
+
+Cloud Custodian not only identifies policy violations but also automates the remediation process. For example, if a policy violation is identified, such as an unsecured storage bucket in AWS, Cloud Custodian can automatically take corrective actions, such as deleting the unsecured bucket or encrypting it.
+
+### Continuous Compliance
+
+Cloud Custodian helps organizations maintain continuous compliance by ensuring that policies are enforced at all times. The tool can detect any changes in the cloud infrastructure that may violate the policies and take corrective actions in real-time.
+
+### Cost Optimization
+
+Cloud Custodian also helps organizations optimize cloud costs by automating the deletion of unused resources, enforcing tagging policies to identify unused resources, and providing reports on cost savings.
+
+## The Watchdog: Snyk
+
+[Snyk](https://snyk.io/){:target="_blank" rel="noopener noreferrer"} is a cloud security platform that helps you to get end-to-end insight into your security footprint.
+
+<img src="{{ '/img/2023-02-16-secure-cloud-foundation/snyk.png' | prepend: site.baseurl }}" alt="Gandalf, You shall not pass" class="image fit" style="margin:0px auto; max-width:100%; height:60%">
+
+### Code scanning
+
+Snyk can scan your code for vulnerabilities and compliance issues.
+It can scan your code for security issues like sql injection or path traversal vulnerabilities.
+
+You might be thinking that nice but how do I figure out how to solve it?
+They have a data flow that shows you the entry point and method invocation of the vulnerability.
+Which shows you the entire stack path to the vulnerability in your source code.
+This helps you in assessing what the impact of a vulnerability is and how urgent you need to provide a patch for it.
+(method that is publicly accessible vs library code that isn't used)
+
+To help you further even more with solving the vulnerability you can read the details of the vulnerability
+and a best practice for preventing it, if available.
+If you thought that would be enough they have another tab that shows 3 open source projects that had the vulnerability
+and how they fixed it in their code base.
+
+### Licence scanning
```

Comment:
Not sure how I feel about this. It's technically correct, but the one thats mostly used (and of course, American :) ) is 'license'. 

Rethink it but I don't have a preferable option here. Do as you wish :)

Reply:
If you decide to change it, change it for the whole article please.

Reply:
Since the documentation of Trivy uses American English, I'll change it. nice spot.